gulpfile.js

```diff
@@ -0,0 +1,872 @@
+                identifiers.length = 0;
+            }
+
+            for (i = 0; i < identifiers.length; ++i) {
+                if (identifiers[i].indexOf('//') >= 0 || identifiers[i].indexOf('/*') >= 0) {
+                    console.log(file + ' contains comments in the require list.  Skipping so nothing gets broken.');
+                    return;
+                }
+            }
+
+            var requires = [];
+
+            for (i = preserveFirst ? 1 : 0; i < names.length && i < identifiers.length; ++i) {
+                requires.push({
+                    name : names[i].trim(),
+                    identifier : identifiers[i].trim()
+                });
+            }
+
+            requires.sort(function(a, b) {
+                var aName = a.name.toLowerCase();
+                var bName = b.name.toLowerCase();
+                if (aName < bName) {
+                    return -1;
+                } else if (aName > bName) {
+                    return 1;
+                } else {
+                    return 0;
+                }
+            });
+
+            if (preserveFirst) {
+                requires.splice(0, 0, {
+                    name : names[0].trim(),
+                    identifier : identifiers[0].trim()
+                });
+            }
+
+            // Convert back to separate lists for the names and identifiers, and add
+            // any additional names or identifiers that don't have a corresponding pair.
+            var sortedNames = requires.map(function(item) {
+                return item.name;
+            });
+            for (i = sortedNames.length; i < names.length; ++i) {
+                sortedNames.push(names[i].trim());
+            }
+
+            var sortedIdentifiers = requires.map(function(item) {
+                return item.identifier;
+            });
+            for (i = sortedIdentifiers.length; i < identifiers.length; ++i) {
+                sortedIdentifiers.push(identifiers[i].trim());
+            }
+
+            var outputNames = ']';
+            if (sortedNames.length > 0) {
+                outputNames = '\r\n        ' +
+                              sortedNames.join(',\r\n        ') +
+                              '\r\n    ]';
+            }
+
+            var outputIdentifiers = '(';
+            if (sortedIdentifiers.length > 0) {
+                outputIdentifiers = '(\r\n        ' +
+                                    sortedIdentifiers.join(',\r\n        ');
+            }
+
+            contents = result[1] +
+                       outputNames +
+                       result[4].replace(/^[,\s]+/, ', ').trim() +
+                       outputIdentifiers +
+                       ') {' +
+                       result[6];
+
+            fs.writeFile(file, contents, callback);
+        });
+    }, done);
+});
+
+function combineCesium(debug, optimizer, combineOutput) {
+    return requirejsOptimize({
+        wrap : true,
```

Comment:
They [still claim](https://github.com/jrburke/r.js/blob/master/build/example.build.js) the default is 0, but obviously that's not the case.  Manually setting the logLevel back to 0 gets us back to the old output.

My one argument for keeping it as is would be to reduce the amount of junk in the travis log file, so that if travis fails you don't have to scroll forever to figure out why.  I did a quick experiment where I changed `createBoxGeometry` to misspell `define` and then ran combine, the build fails as soon as it his the problem and splits our a callstak with the below error on top.

`Error: Error: Parse error using esprima for file: C:/Git/AnalyticalGraphicsInc/cesium/Source/Workers/createBoxGeometry.js`

Here's an example of a successful travis in [this branch](https://travis-ci.org/AnalyticalGraphicsInc/cesium/builds/86845055) compared to another with the [current process](https://travis-ci.org/AnalyticalGraphicsInc/cesium/builds/85309778).

Ultimately, I don't feel that strongly about it, so if you want me to just set it back to the old loglevel of 0, I can do that.  Just let me know either way.